### PR TITLE
add org.kde.tellico

### PR DESCRIPTION
**Pull request by astrorobby**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.kde.tellico.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "id": "org.kde.tellico",
+    "command": "tellico",
+    "branch": "stable",
+    "default-branch": "stable",
```
Comment (hfiguiere) [inline org.kde.tellico.json:5]:
```suggestion
```

You don't need these.

File: org.kde.tellico.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "tellico",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.tellico.json:15]:
```suggestion
        "--socket=fallback-x11",
```

**Comment by hfiguiere** [comment]
bot, build org.kde.tellico

**Comment by barthalion** [comment]
bot, build org.kde.tellico

**Comment by barthalion** [comment]
cc @flathub/kde

**Comment by barthalion** [comment]
The appdata fails validation:

```
builddir/files/share/appdata/org.kde.tellico.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
```

**Comment by hfiguiere** [comment]
bot, build org.kde.tellico

**Pull request closed without merging**